src/main/features/ccj/views/date-of-birth.njk

```diff
@@ -7,7 +7,7 @@
 {% block content %}
   <div class="grid-row">
     <div class="column-two-thirds">
-      <form method="post" novalidate>
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="CCJ: Select defendant date of birth" data-event-label-from="known" novalidate>
```

Comment:
if this is how every form will look like why don't we extract it to macro

```
form(actionName, labelFrom)
```

Reply:
For now only selected forms (these with radio buttons) will submit these custom GA events. Not quite sure how it will be in the future. I treat it a bit like experimental feature. When widely adopted we can thing to make it more reusable.

Also using macro is not handy as we would need to do:

```
{{ form(actionName, labelFrom) }}

</form>
```

Reply:
Aren't you on holidays!? ;)